<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>perfWindow</class>
 <widget class="QWidget" name="perfWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network performance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loadTrainedNetworkButton">
       <property name="text">
        <string>Load trained Network</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="trainedNetworkLabel">
       <property name="text">
        <string>File :</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="plotLossButton">
       <property name="text">
        <string>Plot Loss</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="plotAccuracyButton">
       <property name="text">
        <string>Plot Accuracy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="accuracyF1Button">
       <property name="text">
        <string>Accuracy F1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="plotConfusionMatrixButton">
       <property name="text">
        <string>Plot Confusion Matrix</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
